test/Microsoft.Framework.CommonTestUtils/TestUtils.cs

```diff
@@ -386,5 +386,30 @@ private static bool IsWindows()
             return PlatformHelper.IsWindows;
 #endif
         }
+
+        private static string RemoveAnsiColorCodes(string text)
+        {
+            int escapeIndex = 0;
+            for (; ;)
+            {
+                escapeIndex = text.IndexOf("\x1b[", escapeIndex);
+                if (escapeIndex != -1)
+                {
+                    int endIndex = escapeIndex + 3;
+                    while (endIndex != text.Length && text[endIndex] != 'm')
```

Comment:
endIndex wont index if it equals the length of the string, so no 'out of bounds' happens